playbooks/roles/tarsnap/defaults/main.yml

```diff
@@ -47,10 +47,18 @@ TARSNAP_ARCHIVE_NAME: 'backup'
 # Optional snitch to be sent
 TARSNAP_BACKUP_SNITCH: null
 
-# Backups should be daily at most
-TARSNAP_BACKUP_HOUR: '*'
+# Tarsnap Backup settings. This will be translated to
```

Comment:
The header could be more precise by adding 'cron' somewhere, e.g. 'Tarsnap backup cron settings.'